DSCResources/MSFT_xSqlAlias/MSFT_xSqlAlias.psm1

```diff
@@ -27,7 +27,7 @@ function Get-TargetResource
 
     if ($null -ne (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\MSSQLServer\Client\ConnectTo' -Name "$SQLServerName" -ErrorAction SilentlyContinue))
     {
-        $ItemValue = Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\MSSQLServer\Client\ConnectTo' -Name "$SQLServerName";
+        $ItemValue = Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\MSSQLServer\Client\ConnectTo' -Name "$SQLServerName" -ErrorAction SilentlyContinue;
```

Comment:
in what scenario was the error returned here? Wasn't it being filtered by the if statement on line 28?